ExampleTestProject/Assets/PlayFabSDK/Shared/Internal/PlayFabHttp/PlayFabUnityHttp.cs

```diff
@@ -81,7 +81,7 @@ private static IEnumerator SimpleCallCoroutine(string method, string fullUrl, by
                 yield return request.Send();
 #endif
 
-#if Unity_2021_1_OR_NEWER
+#if UNITY_2021_1_OR_NEWER
```

Comment:
```suggestion
#if UNITY_2020_1_OR_NEWER
```
This change actually went into effect with 2020.1, not 2021.1
Reference: https://docs.unity3d.com/2020.1/Documentation/ScriptReference/Networking.UnityWebRequest.Result.html